dap_prinz_green_jobs/pipeline/green_measures/industries/sic_mapper/sic_mapper_utils.py

```diff
@@ -39,6 +39,37 @@
     ]
 )
 
+# get rid of tokens that often
+# appear in the beginning of job ads
+bad_phrases = [
+    "Job Title",
+    "Job Type",
+    "Salary",
+    "Competitive salary",
+    "competitive salary",
+    "Full-time",
+    "full-time",
+    "Full Time",
+    "part-time",
+    "Part-time",
+    "Permanent",
+    "permanent",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+    "Benefits",
+    "Location",
+    "benefits",
+    "location",
+    "Eye Care",
```

Comment:
Are you sure that cleaning out some of these words might not make things worse for data not in the evaluation set? I'm not concerned with most of them.. but can't help but think "eye care" and "benefits" might be crucial words for some company descriptions, even if the removal of them has helped with your particular evaluation dataset. i.e. could you sample some times outside the evaluation set where these words come up and work out whether the trade off is worth it?

Reply:
(not a comment to be addressed in this PR though)